[quant][executorch] Support inception_v4 in examples

jerryzh168 opened the pull request:
Summary: Verified that pt2e quant flow matches the fx flow with executorch backend config

Test Plan:
with-proxy buck2 run executorch/examples/quantization:example -- -m=ic4 --verify

```
[INFO 2023-08-31 16:08:06,923 example.py:77] prepare sqnr: inf
[INFO 2023-08-31 16:08:06,932 example.py:81] quant diff max: 0.0
[INFO 2023-08-31 16:08:06,936 example.py:85] quant sqnr: inf
```

full output: https://www.internalfb.com/intern/paste/P818520579/

Differential Revision: D48889075

kimishpatel reviewed (approved):
Looks good. Accepting for now, but it would be good to add test for this.

jerryzh168 commented:
> Looks good. Accepting for now, but it would be good to add test for this.

sure I can add a test after refactoring test code, right now it is tested in executorch

jerryzh168 committed:
[quant][executorch] Support inception_v4 in examples (#108382)

Summary:
X-link: https://github.com/pytorch/executorch/pull/203


Verified that pt2e quant flow matches the fx flow with executorch backend config

Test Plan:
with-proxy buck2 run executorch/examples/quantization:example -- -m=ic4 --verify

```
[INFO 2023-08-31 16:08:06,923 example.py:77] prepare sqnr: inf
[INFO 2023-08-31 16:08:06,932 example.py:81] quant diff max: 0.0
[INFO 2023-08-31 16:08:06,936 example.py:85] quant sqnr: inf
```

full output: https://www.internalfb.com/intern/paste/P818520579/

Reviewed By: kimishpatel

Differential Revision: D48889075
